software/control_sw/src/blocks/delay.py

```diff
@@ -135,13 +136,13 @@ def get_delay(self, stream):
         :type stream: int
 
         :return: Currently loaded delay, in ADC samples
-        :rtype: int
+        :rtype: int, scale
```

Comment:
Can this docstring actually explain what the scale is?
And since you can only load integer delays, how can this scale possibly be 2**18?